### ISSUE TITLE
install as a devDependency

### ISSUE THREAD

**Issue by mixmix** [feross/funding#5]
hey @feross I installed this in `devDependencies` (https://github.com/ssbc/ssb-server/pull/685), to see how it would behave.

I notice I only see an ad once, when funding is installed, and thenceforth don't see it (e.g. which I install something)

**Questions**:
- putting it in `devDependencies` is a good idea right? 
    - if so, recommend change docs to `npm install funding --save-dev`
    - who would see it anyway if it was installed in production?
- were you really only expecting the ad to be displayed once?

**Comment by mixmix** [comment]
_as an aside, opening a PR for a project I care a lot about called "add funding" has actually generated a bunch of feels :crying_cat_face:_

**Comment by feross** [comment]
> I notice I only see an ad once, when funding is installed, and thenceforth don't see it

I think this is expected behavior. Basically, we only want to show a sponsor message whenever a package that depends on `funding` is installed, e.g. whenever `standard` is installed. Not whenever _anything_ is installed. I think this is what you were asking, but correct me if I am wrong.

I also noticed that even when running `npm install` to install all dependencies (i.e. not specifying a package name), the sponsor message won't show if `standard` is already installed. So, it only shows up for new installations. I think this is the behavior we want.

> putting it in devDependencies is a good idea right?

I'll repost what I wrote in the linked issue. I think you'll want to make this a dependency if you intend for end users to see the sponsor message. Otherwise, it will only be run for folks who are developing on `ssb-server`.
